src/config/layers.json

```diff
@@ -32,5 +32,14 @@
     "date_interval": "day",
     "opacity": 0.3,
     "legend_text": "ModisLST"
+  },
+  "DzudRiskMap": {
+    "title": "Dzud Risk Map",
+    "server_type": "wms",
+    "server_uri": "http://3.136.214.209:8080/geoserver/prism/wms?layers=prism%3ADZUDRM12311",
+    "has_date": true,
```

Comment:
Also, does it really have dates?

Reply:
Dzud is actually already on master. Maybe you haven't refreshed to the latest version?